osbs/cli/main.py

```diff
@@ -190,6 +190,48 @@ def cmd_list_builds(args, osbs):
         tp.render()
 
 
+def make_digests_str(digests):
+    if digests is not None:
+        try:
+            digests_str = "\n".join(["{registry}/{repository}:{tag} {digest}".format(**dig)
+                                     for dig in digests])
+        except (TypeError, KeyError):
+            digests_str = "(invalid value)"
+    else:
+        digests_str = "(unset)"
+
+    if not digests_str:
+        digests_str = "(empty)"
```

Comment:
It's reached if `digests` is an empty list, which is the case of an unsuccessful build, for example. See unit tests.